NOTICE

```diff
@@ -1,3 +1,3 @@
 Chronicle Map
-Copyright (C) 2012, 2016  higherfrequencytrading.com
+Copyright (C) 2012, 2016-2018 Chronicle Software
```

Comment:
It should be just one line "Copyright (C) 2012, 2018 Chronicle Map contributors" (in the headers of all source files as well). Otherwise each PR author should sign CLA *or* be added into this NOTICE file.